internal/lookout/repository/sql_repository.go

```diff
@@ -100,6 +101,7 @@ type JobRow struct {
 	Finished  sql.NullTime    `db:"finished"`
 	Succeeded sql.NullBool    `db:"succeeded"`
 	Error     sql.NullString  `db:"error"`
+	Reason    sql.NullString  `db:"reason"`
```

Comment:
What else do we get with reason?  

Do we also get https://github.com/G-Research/armada/issues/1478 with this issue?

Reply:
yes that was the idea -- any of the failed Causes should show up:         
```
"Error":            0,
"Evicted":          1,
"OOM":              2,
"DeadlineExceeded": 3,
"Preempted": 4, <-- in Dejan's PR
```